addOns/formhandler/src/main/javahelp/org/zaproxy/zap/extension/formhandler/resources/help/contents/FormHandlerHelp.html

```diff
@@ -58,31 +58,31 @@ <H3>Adding a New Field</H3>
 
 		<H3>Modifying an Existing Field</H3>
 			<p>
-				The add-on allows the user to modify the values and names of existing fields. This can be done 
+				The add-on allows the user to modify the values and names of existing fields. This can be done
 				by selecting the field that you wish to modify and clicking on the <b>Modify</b> button.
-			</p>
-			<img src="../contents/images/formHandlerModDialog.PNG" alt="Image Not Available"/>
+			</p><br>
+			<img src="../contents/images/formHandlerModDialog.PNG" alt="A screenshot of the Form handler's modify dialog"/>
 
 			<p>
 				When modifying a field the user is constrained by the same rules as creating a new one. The most
 				common constraint when modifying a field is that the name cannot be the same as an existing field.
 				(i.e. No duplicate fields)
-			<p>
- 
+			/<p>
```

Comment:
The / needs to be removed as well.

Reply:
It was meant to be inside the <. I just bungled it rushing it seems.